.github/workflows/deployment-ci.yml

```diff
@@ -80,8 +80,3 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: -x test publish
-
-# Disabled for now
-#
-#      - name: Close and Release Repository
-#          run: ./gradlew closeAndReleaseRepository
```

Comment:
@HopeBaron why was this disabled?

Reply:
This was added (and already disabled) by @BartArys in #154, so you might have to ask them.

Reply:
Hello, it's because we wanted to remove the manual part of the work, so release on GH = release on nexus
currently we publish to nexus and I approve

Reply:
I asked Bart to disable it so we can fix the CI and I never turned it back on since then
We can enable it.

Reply:
Now I'm confused. [Here](https://discord.com/channels/556525343595298817/587324906702766226/1042753093781164062) you said we can remove the plugin but here on GH you said we should enable the task (it's from the very same plugin).

Reply:
Yes I didn't update you on this one here.
since we sue the maven-publish plugin now I'm looking through the docs for an alternative.
we can remove the plugin because we are not using it anymore